### PR TITLE
Auto Update Deployer Address

### PR DESCRIPTION
**Pull request by amessbee**
Auto Update Deployer Address in `contract/Makefile`

### CODE REVIEW

**Review by Jovonni — COMMENTED** [review]
You may want to keep this PR small as well, as its easier to follow whats being done

**Comment by Jovonni** [comment]
also `unit` is not passing here as well

**Comment by amessbee** [comment]
@Jovonni cleaned up!

File: contract/Makefile [hunk 1]
```diff
@@ -6,9 +6,8 @@ BLD=000000ubld
 ATOM_DENOM=ibc/BA313C4A19DFBF943586C0387E6B11286F9E416B4DD27574E6909CABE0E342FA
 ATOM=000000$(ATOM_DENOM)
 
-# your deployer address made from make add-address
-ADDR=agoric1vck7hw7wak2y7vv3pnf7am6kmgl7nwgz3ltsla
-# ADDR=agoric125d8s0tamx50gpqdd3m7f5gtkzkcnjs0ehsujr
+# fetch the address of the deployer account from the chain - you may need to adjust this to match your setup
+ADDR=$(kubectl exec -i agoriclocal-genesis-0 -c validator -- agd keys add user1 | jq -r '.address')
```
Comment (Jovonni) [inline contract/Makefile:10]: nice one liner, maybe add $USER or something to the `agd keys add user1`, as sometimes I do this for "alice", and sometimes for "user1"

WDYT?

**Review by Jovonni — APPROVED** [review]
nice one liner 🚀

File: contract/Makefile (hunk 1 above)
Comment (amessbee) [reply contract/Makefile]: Done.

Also, I added a check to see if a user has already been added - in case one calls `make` multiple times.

**Pull request merged**